### ISSUE TITLE
Киберкотлетки. Инпут-компонент с вводом пароля. Некорректное поведение переключения фокусировки через tab

### ISSUE THREAD

**Issue by blackHATred** [VK-BMSTU-QA/homework-2-autumn-2024#43]
### **Страницы, на которых используется затронутый багом компонент ввода пароля:**
[Страница регистрации](https://kinoskop.online/register)
[Страница авторизации](https://kinoskop.online/login)
[Страница настроек профиля](https://kinoskop.online/profile/settings)

### **Как воспроизвести**:
Нажимать клавишу tab, пока окно браузера не установит фокусировку на компоненте ввода пароля.

### **Ожидаемое поведение**:
Сначала фокусировка переходит на поле ввода пароля, затем на кнопку с иконкой глаза.
- Первое нажатие tab:  
  <img width="377" alt="image" src="https://github.com/user-attachments/assets/1f06b331-0fe8-49da-ba97-50ccd07173ca">
- Второе нажатие tab:  
  <img width="369" alt="image" src="https://github.com/user-attachments/assets/776a5b6b-02fe-44e4-9ee0-d19e0ee3e19d">

### **Фактические поведение**:
Сначала фокусировка переходит на кнопку с иконкой глаза, затем на поле ввода пароля.
- Первое нажатие tab:  
  <img width="369" alt="image" src="https://github.com/user-attachments/assets/776a5b6b-02fe-44e4-9ee0-d19e0ee3e19d">
- Второе нажатие tab:  
  <img width="377" alt="image" src="https://github.com/user-attachments/assets/1f06b331-0fe8-49da-ba97-50ccd07173ca">

**Comment by KristinaBu** [comment]
Кристина Буйдина - аппрув

**Comment by tUnknownLegend** [comment]
Давайте название по кароче. На дашборде в джире у вас столько букв не поместится)

**Comment by blackHATred** [comment]
> 2. У вас во всех багах установлен приоритет низкий, но здесь кажется он должен быть ниже чем в других, проблема прям очень мало аффектит юзеров. Не хотите пониже поставить?

Низкий приоритет и так считается самым низким. Выше по порядку идут minor, major и critical. По большому счёту бОльшая часть найденных нами багов - low priority и все они аффектят узкий спектр юзеров; и даже если баг всё-таки случился, то это не мешает выполнению поставленной цели

**Comment by tUnknownLegend** [comment]
> > 2. У вас во всех багах установлен приоритет низкий, но здесь кажется он должен быть ниже чем в других, проблема прям очень мало аффектит юзеров. Не хотите пониже поставить?
> 
> Низкий приоритет и так считается самым низким. Выше по порядку идут minor, major и critical. По большому счёту бОльшая часть найденных нами багов - low priority и все они аффектят узкий спектр юзеров; и даже если баг всё-таки случился, то это не мешает выполнению поставленной цели

Ок, часто используют более широкую классификацию багов, поэтому решил уточнить почему так решили

**Comment by tUnknownLegend** [comment]
> Низкий приоритет и так считается самым низким

Да, окей, если у вас по шкале это самый низкий, то ок. Minor и низкий как будто одно и тоже. Как по мне для удобства лучше использовать вот такую модель, p1-p5 баги, чтобы не путаться, но тут уже как вам удобнее
![image](https://github.com/user-attachments/assets/807092cb-2209-4e1e-b980-acce393b9953)

**Comment by tUnknownLegend** [comment]
принято
